common/persistence/persistence-tests/executionManagerTest.go

```diff
@@ -5149,15 +5149,15 @@ func (s *ExecutionManagerSuite) TestCreateGetUpdateGetShard() {
 	currentReplicationAck := int64(27)
 	currentClusterTransferAck := int64(21)
 	alternativeClusterTransferAck := int64(32)
-	currentClusterTimerAck := timestampConvertor(time.Now().Add(-10 * time.Second))
-	alternativeClusterTimerAck := timestampConvertor(time.Now().Add(-20 * time.Second))
+	currentClusterTimerAck := timestampConvertor(time.Now().UTC().Add(-10 * time.Second))
```

Comment:
Do we want to do a follow up change to continue migration to the timestamp package?

Reply:
Yes, there is a task for it. Let's get rid of ints everywhere where it makes sense.